05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,48 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.container {
+  height:350px;
+  /*this fixes the container height*/
 }
-
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+.berryimage{
+  width:500px;
+  float:left;
 }
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.berryimage img {
+  width:100%;
+  height:100%;
+  /*this fills the whole div with the image */
 }
-
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.berrycontent {
```

Comment:
I don’t believe so.

The float keeps it side by side. So if you take that out then they sit next to each other.

If you take the width out then the width will change to the content that it in it div. By having it 500px it keeps it uniform. 


Haley Grace
hcgrace.05@gmail.com



> On May 16, 2016, at 1:26 PM, Victoria Chuang <notifications@github.com> wrote:
> 
> In 05LessonFive/Checkpoint1/css/style.css <https://github.com/AustinCodingAcademy/intro-workbook/pull/6#discussion_r63400486>:
> 
> >  }
> > -
> > -.left {
> > -    float: left;
> > -    width: 60%;
> > -    height: 300px;
> > -    margin-left: 260px;
> > -    background-color: lightgray;
> > +.berrycontent {
> if both berryimage and berrycontent have the same width and float, could you factor these out?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/AustinCodingAcademy/intro-workbook/pull/6/files/2687aa593ab3254c26d7d4fa8ce74887c5b4776c#r63400486>